cloudinary-core/src/main/java/com/cloudinary/Transformation.java

```diff
@@ -797,6 +797,15 @@ private static String normRangeValue(Object objectValue) {
         return matcher.group(1) + modifier;
     }
 
+    private static String normAutoRangeValue(Object objectValue) {
+        if (objectValue == null) return null;
```

Comment:
You don't really need a null check here, using `"auto".equals(objectValue)` is enough, if true return `objectValue.toString()`, otherwise continue on to `normRangeValue`.
This can be important in case we decide to handle null values differently, and then we won't have to fix it in both places.